### PR TITLE
Merge TAP Beta to V1.0

### PR DESCRIPTION
**Pull request by tilarso**
updating docs to reflect TAP release from Beta to V1.0 

Schema update PR 6073317

### CODE REVIEW

File: api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-get.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    "isUsableOnce": false,
+    "includeTargets": [
+        {
+            "targetType": "group",
+            "id": "all_users",
+            "isRegistrationRequired": false
+        }
+    ]
+}
+```
```
[the remark this answers is not in the document]
Comment (tilarso) [reply api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-get.md:79]: Got it. Will fix in the v1.0 doc.

File: api-reference/v1.0/api/temporaryaccesspassauthenticationmethod-get.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  "blockType": "request",
+  "name": "get_temporaryAccessPassAuthenticationMethod"
+}
+-->
+```msgraph-interactive
+GET https://graph.microsoft.com/v1.0/users/071cc716-8147-4397-a5ba-b2105951cc0b/authentication/temporaryAccessPassMethods/05267842-25b2-4b21-8abd-8e4982796f7f
+```
+
+### Response
+**Note:** The response object shown here might be shortened for readability.
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/temporaryaccesspassauthenticationmethod-get.md:78]:
```suggestion
>**Note:** The response object shown here might be shortened for readability.
```

File: api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-get.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}. Required.|
+
+## Request body
+Do not supply a request body for this method.
+
+### Response
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-get.md:47]:
```suggestion
## Response
```

File: api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-get.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+}
+-->
+```msgraph-interactive
+https://graph.microsoft.com/v1.0/policies/authenticationMethodsPolicy/authenticationMethodConfigurations/temporaryAccessPass
+```
+
+### Response
+
+The following is an example of the response
+**Note:** The response object shown here might be shortened for readability.
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-get.md:65]:
```suggestion
>**Note:** The response object shown here might be shortened for readability.
```

File: api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-update.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+### Request
+<!-- {
+  "blockType": "request",
+  "name": "update_temporaryaccesspassauthenticationmethodconfiguration"
+}
+-->
+``` http
+PATCH https://graph.microsoft.com/v1.0/temporaryAccessPassAuthenticationMethodConfiguration
+Content-Type: application/json
+{
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-update.md:68]: The spacing's important. Will auto-resolve 1 API Doctor error
```suggestion

{
```

File: api-reference/v1.0/resources/temporaryaccesspassauthenticationmethod.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+|[Delete](../api/temporaryaccesspassauthenticationmethod-delete.md)|None|Delete a user's **temporaryAccessPassAuthenticationMethod** object.|
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|createdDateTime|DateTimeOffset|The date and time when the Temporary Access Pass was created.|
+|id|String|The identifier of the Temporary Access Pass registered to this user. Inherited from [entity](../resources/entity.md).|
+|isUsableOnce|Boolean|Determines whether the pass is limited to a one-time use. If `true`, the pass can be used once; if `false`, the pass can be used multiple times within the Temporary Access Pass lifetime.|
+|isUsable|Boolean|The state of the authentication method that indicates whether it's currently usable by the user.|
+|lifetimeInMinutes|Int32|The lifetime of the Temporary Access Pass in minutes starting at startDateTime. Must be between 10 and 43200 inclusive (equivalent to 30 days).|
```
Comment (FaithOmbongi) [inline api-reference/v1.0/resources/temporaryaccesspassauthenticationmethod.md:32]:
```suggestion
|lifetimeInMinutes|Int32|The lifetime of the Temporary Access Pass in minutes starting at **startDateTime**. Must be between 10 and 43200 inclusive (equivalent to 30 days).|
```

File: api-reference/v1.0/api/authentication-post-temporaryaccesspassmethods.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+|Property|Type|Description|
+|:---|:---|:---|
+|isUsableOnce|Boolean|Optional. Determines if the pass is limited to a one-time use. If `true`, the pass can be used once; if `false`, the pass can be used multiple times within its **lifetimeInMinutes** setting. A multi-use Temporary Access Pass (`isUsableOnce = false`), can only be created and used for sign-in if it is allowed by the  [Temporary Access Pass authentication method policy](temporaryaccesspassauthenticationmethodconfiguration.md).|
+|lifetimeInMinutes|Int32|Optional. The lifetime of the temporaryAccessPass in minutes starting at creation time or at startDateTime, if set. Must be between 10 and 43200 (equivalent to 30 days). If not specified, the **defaultLifetimeInMinutes** setting in the [Temporary Access Pass authentication method policy](temporaryaccesspassauthenticationmethodconfiguration.md) is applied. |
+|startDateTime|DateTimeOffset|Optional. The date and time when the temporaryAccessPass becomes available to use. If not specified, the Temporary Access Pass is available to use immediately after it's created.|
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/authentication-post-temporaryaccesspassmethods.md:57]: Link fixes, will auto-resolve 2 API Doctor errors
```suggestion
|isUsableOnce|Boolean|Optional. Determines if the pass is limited to a one-time use. If `true`, the pass can be used once; if `false`, the pass can be used multiple times within its **lifetimeInMinutes** setting. A multi-use Temporary Access Pass (`isUsableOnce = false`), can only be created and used for sign-in if it is allowed by the  [Temporary Access Pass authentication method policy](../resources/temporaryaccesspassauthenticationmethodconfiguration.md).|
|lifetimeInMinutes|Int32|Optional. The lifetime of the temporaryAccessPass in minutes starting at creation time or at startDateTime, if set. Must be between 10 and 43200 (equivalent to 30 days). If not specified, the **defaultLifetimeInMinutes** setting in the [Temporary Access Pass authentication method policy](../resources/temporaryaccesspassauthenticationmethodconfiguration.md) is applied. |
|startDateTime|DateTimeOffset|Optional. The date and time when the temporaryAccessPass becomes available to use. If not specified, the Temporary Access Pass is available to use immediately after it's created.| 
```

File: api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-get.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+## Examples
+
+### Request
+<!-- {
+  "blockType": "request",
+  "name": "get_temporaryaccesspassauthenticationmethodconfiguration"
+}
+-->
+```msgraph-interactive
+https://graph.microsoft.com/v1.0/policies/authenticationMethodsPolicy/authenticationMethodConfigurations/temporaryAccessPass
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-get.md:59]: Missing method. Will auto-resolve API Doctor error
```suggestion
GET https://graph.microsoft.com/v1.0/policies/authenticationMethodsPolicy/authenticationMethodConfigurations/temporaryAccessPass
```

File: api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-get.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+```
+
+### Response
+
+The following is an example of the response
+**Note:** The response object shown here might be shortened for readability.
+
+```http
+Content-Type: application/json
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-get.md:68]: Missing protocol info
```suggestion
HTTP/1.1 200 OK
Content-Type: application/json
```

File: api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-get.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+```msgraph-interactive
+https://graph.microsoft.com/v1.0/policies/authenticationMethodsPolicy/authenticationMethodConfigurations/temporaryAccessPass
+```
+
+### Response
+
+The following is an example of the response
+**Note:** The response object shown here might be shortened for readability.
+
+```http
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-get.md:67]: Missing HTML comment block - will auto-resolve 1 API Doctor error
```suggestion
<!-- {
  "blockType": "response",
  "truncated": true,
  "@odata.type": "microsoft.graph.temporaryAccessPassAuthenticationMethodConfiguration"
}
-->
```http
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @tilarso, Requested some minor fixes here and there, pointed out some that relate to the [API Doctor errors](https://dev.azure.com/msrobots/apidoctor/_build/results?buildId=85081&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=dac0b392-2308-5704-b9f7-c5b20adf012e&l=24691) logged against your PR.

I've also pushed a commit to fix some sections in the GET policy and LIST method `beta` docs. I couldn't leave a comment in those sections.

File: api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-update.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+## Examples
+
+### Request
+<!-- {
+  "blockType": "request",
+  "name": "update_temporaryaccesspassauthenticationmethodconfiguration"
+}
+-->
+```http
+PATCH https://graph.microsoft.com/v1.0/temporaryAccessPassAuthenticationMethodConfiguration
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-update.md:66]: Hi @tilarso - This line could be the cause of the [API doctor error](https://dev.azure.com/msrobots/apidoctor/_build/results?buildId=85415&view=logs&j=92a4755a-6173-591e-5c07-8dbe1165e937&t=c54bce61-9f10-5a6f-8d73-72139541958d&s=d654deb9-056d-50a2-1717-90c08683d50a).
```suggestion
PATCH https://graph.microsoft.com/v1.0/policies/authenticationMethodsPolicy/authenticationMethodConfigurations/temporaryAccessPass
```

File: api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-update.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}. Required.|
+|Content-Type|application/json. Required.|
+
+## Request body
+In the request body, supply a JSON representation of the [temporaryAccessPassAuthenticationMethodConfiguration](../resources/temporaryaccesspassauthenticationmethodconfiguration.md) object with the values of fields that should be updated. Existing properties that are not included in the request body will maintain their previous values or be recalculated based on changes to other property values. For best performance, don't include existing values that haven't changed.
+
+All properties of the object can be updated. For a list of properties, see [temporaryAccessPassAuthenticationMethodConfiguration](../resources/temporaryaccesspassauthenticationmethodconfiguration.md).
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/temporaryaccesspassauthenticationmethodconfiguration-update.md:48]:
```suggestion
All properties and relationships of the object can be updated. For the list of properties and relationships, see [temporaryAccessPassAuthenticationMethodConfiguration](../resources/temporaryaccesspassauthenticationmethodconfiguration.md).
``

**Review by FaithOmbongi — APPROVED** [review]
Hi @tilarso, I've left 2 additional suggestions to close minor gaps. Approved with suggestions.

**Pull request merged**